### ISSUE TITLE
Stellaris - Steel Ore cannot be broken when mining it

### ISSUE THREAD

**Issue by eduardo911** [FTBTeam/FTB-Modpack-Issues#6454]
### Modpack

FTB Presents Direwolf20 1.21

### Modpack version

1.8.2

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

Stellaris raw Steel Ore cannot be mined using pickaxes. It will take long to actually break it and it will not yield any ore back.

### Steps to reproduce

1. find any stellaris steel ore
2. try to mine it

### Expected behaviour

will take a really long time to break and it won't yield any ore

**Comment by ArtpokeBr** [comment]
Hmm just out of curiosity, where did you find a Steel Ore? they should be disabled.

**Comment by eduardo911** [comment]
> Hmm just out of curiosity, where did you find a Steel Ore? they should be disabled.
> 
> 
> 
> But just to make sure that people manage to find a way to get them, you should be able to get Raw Ores from it now and Blast Furnace them into Steel Ingots.

I've been finding it throughout the world, perhaps I loaded the world before it was disabled? I can see that dark steel requires blast furnace.

**Comment by ArtpokeBr** [comment]
yeah might be due to that, we technically have the Ore Disabled so in new chunks they shouldn't generate, please if you don't mind, could you test this for me? :)

Also like I've mentioned before, you shouldn't find it, but in the next update I've added back the tags for it to be mineable and recipes to process the raw ore that are somewhat balanced.
